src/material/button/button-base.ts

```diff
@@ -95,6 +103,7 @@ export class MatButtonBase
     elementRef: ElementRef,
     public _platform: Platform,
     public _ngZone: NgZone,
+    private _focusMonitor: FocusMonitor,
```

Comment:
Should we use the focus monitor styles for the focus styles?

Reply:
Are you referring to these styles here: https://github.com/angular/components/blob/main/src/material/legacy-button/_button-base.scss#L67 I believe that should be the only one for the button component. Visually I can't tell the difference between having those styles or not, but maybe they were included with the legacy button for a reason.

Reply:
Those are the focus styles for the legacy button. I was referring to these https://github.com/angular/components/blob/main/src/material/button/_button-theme-private.scss#L21.

Reply:
Got it, added `.cdk-focused` - would that suffice?

Reply:
I think that we should match the behavior of the old button where the focus indication is only shown for keyboard and programmatic focus. Also the `:focus` selector is now redundant and can be removed.

Reply:
Makes sense - done.